reco_utils/dataset/pandas_df_utils.py

```diff
@@ -402,3 +403,74 @@ def has_same_base_dtype(df_1, df_2, columns=None):
             result = False
 
     return result
+
+
+class PandasHash:
```

Comment:
Really should inheret from Object https://wiki.python.org/moin/NewClassVsClassicClass

Reply:
oh, we're only supporting python 3.x so there are only new style classes here